templates/typegen/types.go.tmpl

```diff
@@ -42,6 +42,18 @@ type {{.Name}} struct {
   {{    .Name }} {{ .Type }} {{ .Tags }}
   {{- end}}
 }
+{{-  if .GenerateGetters }}
+{{-   range .Fields }}
+// Get{{ .Name }} returns a pointer to the value of {{ .Name }} from {{ $typeName }} or nil
+func (x *{{ $typeName }}) Get{{ .Name}}() *{{ .Type }} {
```

Comment:
Like I said, this feels a bit odd to return a pointer.  My reasoning was two fold.  Gotta nil check the receiver and return something (empty value or what not), and decrease creation of a ton of objects in memory for little gain.

Issues of course would be that you could change the value, which could lead to some pretty hard to find bugs...

Alternatives could be to return the value and an error / nil.  Which is arguably the best option now that I'm typing this comment...